cmd/instances.go

```diff
@@ -0,0 +1,254 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"strings"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/cozy/cozy-stack/config"
+	"github.com/cozy/cozy-stack/crypto"
+	"github.com/cozy/cozy-stack/instance"
+	"github.com/howeyc/gopass"
+	"github.com/spf13/cobra"
+)
+
+var flagLocale string
+var flagApps []string
+
+// instanceCmdGroup represents the instances command
+var instanceCmdGroup = &cobra.Command{
```

Comment:
[nitpick] we are already in the "instances" file, there is no need to put the word instance in all private variable names.